src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -706,10 +703,10 @@ Json::Value checkFee (
 
     if (fee > limit && fee != loadFee &&
         request.isMember("x-queue-okay") &&
-            request["x-queue-okay"].isBool() &&
+            request["x-queue-okay"].isConvertibleTo(Json::booleanValue) &&
```

Comment:
What convinced me is that I just realized that Json strings are actually NOT convertible to 'bool' according to `isConvertibleTo`. https://github.com/ximinez/rippled/blob/ca09d2844c7f7385ad5b228e003cb809a6e55f73/src/ripple/json/impl/json_value.cpp#L722-L724